packages/keystone/lib/ListTypes/list.js

```diff
@@ -326,7 +326,6 @@ module.exports = class List {
   }
 
   getAdminMeta({ schemaName }) {
-    const schemaAccess = this.access[schemaName];
```

Comment:
Most of `list.getAdminMeta` has been removed.

Reply:
this `list.getAdminmeta` may be useful to wrap few things from new provider, can it be?

Reply:
This is server-side. What were you thinking of putting here?

Reply:
ah, I missed, there is another list class in client side as well.

I guess you did not reuse the getAdminMeta to populate the resulting gql response.

Reply:
Indeed. That is handled by a combination of `list.listMeta` and the CRUD provider.